src/views/DashboardView.vue

```diff
@@ -1,12 +1,18 @@
 <template>
-  <div class="about">
-    <h1>This is an Dashbord Page</h1>
+  <Navbar></Navbar>
+  <div class="m-5">
+    <Filter></Filter>
+    <ProductCard></ProductCard>
```

Comment:
Product should be for rendering a single card not list of cards use for here to render it multiple times.